src/lib/comms/sol-lwm2m.c

```diff
@@ -4809,6 +4813,22 @@ bootstrap_finish(void *data, struct sol_coap_server *coap,
 
     r = sol_coap_send_packet(coap, response, cliaddr);
     dispatch_bootstrap_event_to_client(client, SOL_LWM2M_BOOTSTRAP_EVENT_FINISHED);
+    //TODO: delete connection with bs-server after bs-finish, this is conflicting with the
+    // code in get_server_id_by_link_addr as well
+///*
+    SOL_WRN("FINISH:\ncliaddr=%s", sol_network_link_addr_to_str(cliaddr, &addr));
+    SOL_VECTOR_FOREACH_IDX (&client->connections, conn_ctx, i) {
+        server_addr = sol_vector_get_no_check(&conn_ctx->server_addr_list, conn_ctx->addr_list_idx);
+        SOL_WRN("server_addr=%s", sol_network_link_addr_to_str(server_addr, &addr));
```

Comment:
You should not use this as WRN. You can change to DBG, however add more information about it